session.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 // The ngrok library version.
-const libraryAgentVersion = "3.0.0-ngrok-go"
+const libraryAgentVersion = "0.0.0"
```

Comment:
Should we leave in `+golang` still? Seems like something that'll be handy to have in the version, unless there's a more obvious place it is session-level?

Reply:
Yeah, the version isn't the place to put it. Semver-wise, it's a version tag that just marks it as "incompatible" iirc.

Another field like "library id" would be the more correct approach. I even had it over in #13827 before deciding that I didn't really have a use for it yet, and so omitted it.

Reply:
The semver spec calls the stuff after the `+` "Build metadata", which doesn't seem that wrong of a place to put the language.

I agree having more properly structured fields would be better. It does seem like something worth having before a general release though since I think we'll certainly want metrics on language client usage, and also want that to be visible in metrics for identifying any issues with specific clients/versions.